platforms/nuttx/src/px4/nxp/imxrt/adc/adc.cpp

```diff
@@ -186,6 +186,12 @@ uint32_t px4_arch_adc_sample(uint32_t base_address, unsigned channel)
 	result  = rR0(base_address);
 	return result;
 }
+
+float px4_arch_adc_reference_v()
+{
+	return 3.3f;	// TODO: provide true vref
```

Comment:
Please define 
BOARD_ADC_POS_REF_V 3.3 in board_common.h.  Then use that define for all of these in every adc driver. Then in the board_config.h for boards that do not have a 3.3 ref Add the define there to override the default. 

create a devcollapse the #defines in all the  in  use the define in board_config.h - It sets the wiring.

Reply:
Done with 047f400c90